docs/DeveloperGuide.adoc

```diff
@@ -358,13 +358,290 @@ image::UndoRedoActivityDiagram.png[width="650"]
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two * different things.
 // end::undoredo[]
 
+// tag::orders[]
+=== Orders
+
+Orders refer to the sales orders that will be added by the salesperson to ContactSails.
+These orders can be added to any particular person of the address book.
+After being added, orders can be edited and deleted.
+
+==== Current Implementation
+
+An `Order` is represented in the application as shown in _Figure 11_ given below.
+
+// UML Diagram for Order
+.UML Diagram for Order
+image::OrderUMLDiagram.png[width="500" align="center"]
+
+An `Order` is implemented in the following manner:
+[source,java]
+----
+public class Order {
+
+    private final OrderInformation orderInformation;
```

Comment:
Please use code snippets minimally.
Reason: it is difficult to maintain these in the document, particularly if the implementation changes.